cmd/spire-agent/cli/run/run.go

```diff
@@ -96,6 +96,15 @@ func (*RunCLI) Run(args []string) int {
 		return 1
 	}
 
+	// Create uds dir and parents if not exists
+	dir := filepath.Dir(c.BindAddress.String())
```

Comment:
What would you think stat'ing the directory first before doing the log+mkdirall and only proceeding if the directory does not exist? The logging might be less confusing (I'd be wondering why it was logging that it was creating the directory when I knew it already existed).

Reply:
Ha, that was going to be my next move after I confirmed the code location. Sounds good :+1: